semperfiwebdesign/all-in-one-seo-pack
# Remove version check for Gutenberg/Chrome compatibility fix and check if user agent is set

Issue #2914 & #2952

## Proposed changes

Removes the version check for the Gutenberg/Chrome compatibility fix since the Gutenberg team decided not to implement a fix of their own.

## Types of changes

- Improves existing functionality
- Improves existing code

## Checklist


## Testing instructions

Use a Chromium v77 browse and reproduce the issue; then install this PR and make sure the fix still works.
